### PR TITLE
Prep for devtools_shared 2.3.0 null safe release

### PR DESCRIPTION
**Pull request by kenzieschmoll**

### CODE REVIEW

**Review by jacob314 — APPROVED** [review]
lgtm

**Review by jacob314 — COMMENTED** [review]
lgtm

File: tool/build_release.sh [hunk 1]
```diff
@@ -63,3 +63,7 @@ flutter build web \
 mv build/web ../devtools/build
 
 popd
+
+pushd packages/devtools
```
Comment (jacob314) [inline tool/build_release.sh:67]: did you mean to add these lines to build_release?

Comment (kenzieschmoll) [reply tool/build_release.sh:67]: yep. This was a required step to being able to build and run devtools. This happens as part of the publish script but was missing here.

**Pull request merged**